ios-target-app/Managers/UserDataManager.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UserDataManager.swift
+//  ios-target-app
+//
+//  Created by Raul Piñeres Carrera on 1/11/22.
+//
+
+import Foundation
+
+class UserDataManager: NSObject {
+  
+  static var currentUser: User? {
+    get {
+      let defaults = UserDefaults.standard
+      if
+        let data = defaults.data(forKey: "ios-base-user"),
```

Comment:
pls use a different key to store this value. Also you could move it to a constant so it is less error-prone